src/seedu/addressbook/commands/order/DraftOrderEditPointsCommand.java

```diff
@@ -33,14 +32,20 @@ public DraftOrderEditPointsCommand(int points) {
     @Override
     public CommandResult execute() {
         try {
-            final ReadOnlyMember member = rms.getMemberFromDraftOrder();
-            final int points = toRedeem.getPoints();
-            final ReadOnlyMember emptyMember = new Member();
-            if (member.getName().equals(emptyMember.getName())) {
+            final ReadOnlyOrder draftOrder = rms.getDraftOrder();
+            final ReadOnlyMember customer = draftOrder.getCustomer();
```

Comment:
you should use getMemberFromDraftOrder() instead of draftOrder.getCustomer()

Reply:
See comment above